### PR TITLE
cinemaci/h5py: Extend python needed

### PR DESCRIPTION
**Pull request by kwryankrattiger**
Add `exends('python')` to `py-cinemasci` and `py-h5py`. Without this running cinemasci fails to import h5py and so on.

@chuckatkins

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-cinemasci/package.py [hunk 1]
```diff
@@ -19,10 +19,10 @@ class PyCinemasci(PythonPackage):
 
     version('1.3', sha256='c024ca9791de9d78e5dad3fd11e8f87d8bc1afa5830f2697d7ec4116a5d23c20')
 
+    extends('python')
+
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-cinemasci/package.py:23]: Why is this needed? In principle it should be inherited from the base class.

File: var/spack/repos/builtin/packages/py-h5py/package.py [hunk 2]
```diff
@@ -33,6 +33,8 @@ class PyH5py(PythonPackage):
     version('2.5.0', sha256='9833df8a679e108b561670b245bcf9f3a827b10ccb3a5fa1341523852cfac2f6')
     version('2.4.0', sha256='faaeadf4b8ca14c054b7568842e0d12690de7d5d68af4ecce5d7b8fc104d8e60')
 
+    extends('python')
+
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/py-h5py/package.py:37]: Same here

**Review by alalazo — CHANGES_REQUESTED** [review]

**Comment by chuckatkins** [comment]
Looks like you're right.  I think this got caught up in some other issues we were dealing with.
